packages/cozy-mjml/src/components/MJFooter.js

```diff
@@ -0,0 +1,137 @@
+const validator = require('mjml-validator')
+const core = require('mjml-core')
+
+validator.registerDependencies({
+  'mj-body': ['mj-footer'],
+  'mj-footer': []
+})
+
+class MJFooter extends core.BodyComponent {
+  displayInstance(instance) {
+    if (instance) {
+      return `
+        <mj-section padding="16px 0">
+          <mj-column vertical-align="middle">
+            <mj-button href="https://${instance}" background-color="#fff" font-size="14px" color="#297ef2" text-decoration="none" font-weight="bold" padding="0" inner-padding="4px 16px" vertical-align="middle" border-radius="16px">
+              <img width="24" height="24" src="../src/assets/images/cozy-logo-round.png" style="vertical-align:middle;" />&nbsp;
+              ${instance}
+            </mj-button>
+          </mj-column>
+        </mj-section>`
+    } else {
+      return `
+        <mj-section padding="16px 0 8px">
+          <mj-column align="center">
+            <mj-image padding="0" width="32px" src="../src/assets/images/cozy-logo-round.png"></mj-image>
+          </mj-column>
+        </mj-section>`
+    }
+  }
+
+  render() {
+    if (this.getAttribute('locale') === 'fr') {
```

Comment:
Maybe we could make the code a little bit simpler and avoid repeating markup with something like:
```js
const locales = {
  fr: {
    tagline: "Cozy, votre domicile numérique"  
	// ...
  },
  en: {
    tagline: "Cozy, your smart personal cloud"
    // ...
  } 
}

// ...

class MJFooter extends core.BodyComponent {
  // ...
  render(){
    const lang = this.getAttribute('locale')
    const translations = locales[lang]

    return this.renderMJML(
      `<mj-wrapper padding="0">
       // ...
          <mj-text padding="0" align="center" mj-class="highlight">${translations.tagline}</mj-text>
       // ...
    )
  }
}
```

Reply:
Ok, @nono did it in the next commits 👍